openedx/core/djangoapps/xblock/runtime/blockstore_runtime.py

```diff
@@ -67,9 +67,17 @@ def get_block(self, usage_id, for_parent=None):
                 # When a former XModule no longer needs to support the old runtime, its parse_xml_new_runtime method
                 # should be removed and its parse_xml() method should be simplified to just call the super().parse_xml()
                 # plus some minor additional lines of code as needed.
-                block = block_class.parse_xml_new_runtime(xml_node, runtime=self, keys=keys)
+                try:
+                    block = block_class.parse_xml_new_runtime(xml_node, runtime=self, keys=keys)
+                except Exception:
+                    log.warning('XBlockParseException when parse_xml_new_runtime xblock')
```

Comment:
Can you log the exception as well?  Otherwise there is no way to see which xblocks are broken.